man/chpl.rst

```diff
@@ -136,6 +136,11 @@ OPTIONS
     "unknown tool name" warning. To provide multiple tool names, use one
     **\--using-attribute-toolname** flag per name.
 
+**\--[no-]warn-race-conditions**
+
+    Enable [disable] the printing compiler warnings for potential race
+    conditions. Defaults to not printing race condition warnings.
```

Comment:
This is where it might be nice to say a bit more about what kinds of races are detected.  E.g., is it any `A[B]` on the LHS of an assignment?  Assignment-like operator?  Passed to a `ref` argument?  Etc.